src/event.rs

```diff
@@ -258,9 +266,9 @@ proptest! {
     #[test]
     fn it_should_have_different_hashes_on_different_transactions(tx1 in "[a-z]*", tx2 in "[a-z]*") {
         use crate::event::parents::ParentsPair;
-        let event1: Event<ParentsPair> = Event::new(vec![tx1.as_bytes().to_vec()], None, Vec::new());
-        let event2: Event<ParentsPair> = Event::new(vec![tx2.as_bytes().to_vec()], None, Vec::new());
-        let event3: Event<ParentsPair> = Event::new(vec![tx2.as_bytes().to_vec()], None, Vec::new());
+        let event1: Event<ParentsPair> = Event::new(vec![tx1.as_bytes().to_vec()], Vec::new(), None, Vec::new());
+        let event2: Event<ParentsPair> = Event::new(vec![tx2.as_bytes().to_vec()], Vec::new(), None, Vec::new());
+        let event3: Event<ParentsPair> = Event::new(vec![tx2.as_bytes().to_vec()], Vec::new(), None, Vec::new());
```

Comment:
Can you change them all to `vec![]` XOR `Vec::new()`, the lack of consistency gets to me 😅

Reply:
Changed to vec![] - more compact :)

Reply:
Great, there's still plenty missing though. Try a:
```
fd -t f -e rs -exec sed -i 's/Vec::new()/vec![]/g' {} \;
```

Reply:
changed it globally :)